src/lib/core/tests/TestTLV.cpp

```diff
@@ -1716,35 +1707,35 @@ void CheckOctetStringPrettyPrinter(nlTestSuite * inSuite, void * inContext)
 
     TLVWriter writer;
     writer.Init(encodedBuf);
-    NL_TEST_ASSERT_SUCCESS(inSuite, writer.PutBytes(CommonTag(0), testOctetString, sizeof(testOctetString)));
-    NL_TEST_ASSERT_SUCCESS(inSuite, writer.Finalize());
+    EXPECT_EQ(CHIP_NO_ERROR, writer.PutBytes(CommonTag(0), testOctetString, sizeof(testOctetString)));
+    EXPECT_EQ(CHIP_NO_ERROR, writer.Finalize());
 
     TLVReader reader;
     reader.Init(encodedBuf, writer.GetLengthWritten());
 
     chip::TLV::Debug::Dump(reader, StringDumpWriter);
 
-    NL_TEST_ASSERT(inSuite, strlen(expectedPrint) == strlen(gStringDumpWriterBuf));
-    NL_TEST_ASSERT(inSuite, strcmp(expectedPrint, gStringDumpWriterBuf) == 0);
+    EXPECT_EQ(strlen(expectedPrint), strlen(gStringDumpWriterBuf));
+    EXPECT_EQ(strcmp(expectedPrint, gStringDumpWriterBuf), 0);
```

Comment:
Some instances of https://pigweed.dev/pw_unit_test/#c.EXPECT_STREQ

Maybe we can dump the length compare above since the string compare should cover that.

Reply:
Done